### ISSUE TITLE
[PARSeq] Conversion from ONNX to Tensorflow

### ISSUE THREAD

**Issue by SirNader** [PINTO0309/onnx2tf#245]
### Issue Type

Others

### onnx2tf version number

1.7.24

### onnx version number

1.13.0

### tensorflow version number

2.9.2

### Download URL for ONNX

https://drive.google.com/file/d/1K2NMxEgqv0CqMjlyI06wJtDmzNdMldeN/view?usp=share_link

### Parameter Replacement JSON

```json
{}
```

### Description

1) Research Purpose (University research project)
2) When I attempt to convert the onnx model (parseq.onnx) using this command:
`onnx2tf -i parseq.onnx`
I end up with the following error:

`Cannot reshape a tensor with 43 elements to shape [1,1] (1 elements) for '{{node tf.reshape_81/Reshape}} = Reshape[T=DT_INT64, Tshape=DT_INT32](Placeholder, tf.reshape_81/Reshape/shape)' with input shapes: [43], [2] and with input tensors computed as partial shapes: input[1] = [1,1].`
`Call arguments received by layer "tf.reshape_81" (type TFOpLambda):
  • tensor=tf.Tensor(shape=(43,), dtype=int64)
  • shape=['1', '1']
  • name='/Reshape_1'
`
3) Unfortunately, I tried to follow the parameter replacement guidelines but couldn't resolve my problem.
4) This will help me make a decision on whether to use this model in my research project. I  am working on implementing a ML pipeline using Tensorflow and I need to convert this PyTorch model from ONNX to Tensorflow. 
5) The source code of this model is https://github.com/baudm/parseq

I would be so grateful if you showed me how to deal with this parameter replacement thing.

Thank you!

**Comment by PINTO0309** [comment]
https://github.com/PINTO0309/onnx2tf/releases/tag/1.7.25

https://github.com/PINTO0309/onnx2tf/blob/main/json_samples/replace_parseq.json

**Comment by PINTO0309** [comment]
https://github.com/baudm/parseq/issues/66

**Comment by SirNader** [comment]
@PINTO0309 Thanks Very Much Katsuya!
I really appreciate your help

**Comment by SirNader** [comment]
Hello again @PINTO0309


So I have fixed the onnx model and it worked! and converted to TFLite
However when I compare the results between the ONNX and TFLite, they are different.

models: https://drive.google.com/file/d/1CjzmWjRpzPEioAaJzDkb077MAFnSjEOC/view?usp=sharing
here's the input I used to compare: [input.zip](https://github.com/PINTO0309/onnx2tf/files/10959774/input.zip)
PS: I have did transpose when I did inference on the TFLite model
`numpy_input = np.load('input.npy')`
`numpy_input = np.transpose(numpy_input,(0,2,3,1))`

Any thoughts?

**Comment by PINTO0309** [comment]
The outputs match perfectly.
```
wget https://github.com/PINTO0309/PINTO_model_zoo/raw/main/360_PARSeq/replace_parseq.json

pip install onnx2tf -U
onnx2tf -i parseq.onnx -prf replace_parseq.json
```
![image](https://user-images.githubusercontent.com/33194443/224852077-6dabcf07-2e0b-4194-b881-dde3e89f9142.png)

**Comment by SirNader** [comment]
> The outputs match perfectly. Your onnx used to be broken so you could not perform accuracy checks, but now you can use the `-cotof` option to perform accuracy checks on onnx and tf.
> 
> ```
> wget https://github.com/PINTO0309/PINTO_model_zoo/raw/main/360_PARSeq/replace_parseq.json
> 
> pip install onnx2tf -U
> onnx2tf -i parseq.onnx -prf replace_parseq.json -cotof -rtpo Erf
> ```
> 
> ![image](https://user-images.githubusercontent.com/33194443/224852077-6dabcf07-2e0b-4194-b881-dde3e89f9142.png)

Thank You very much!
